doc/source/building/build_in_buildservice.rst

```diff
@@ -195,3 +216,43 @@ setting is not sufficient. The following additional line is required:
 .. code:: bash
 
    Release: <CI_CNT>.<B_CNT>
+
+If you are building Vagrant images (see :ref:`setup_vagrant`) then also add
```

Comment:
* If you are building Vagrant images -> If you build Vagrant images
* Remove "then also"